### ISSUE TITLE
The 50´s SP Exit is wrongly name as the "right" in the app, when it is actually the "left".

### ISSUE THREAD

**Issue by LautaroL20** [BEEmod/BEE2.4#1811]
### BEE2 Application Version

4.41.0

### BEE2 Package Version

4.41.0

### What operating system are you running on?

Windows 7 (64bit)

### Description of the bug

The window for choosing a SP Exit cooridor shows the name of the 50´s SP Exit corridor as "4: Right Turn", when the true name for this should be "4: Left Turn":

![50s corridor](https://user-images.githubusercontent.com/12244376/152383688-e042eff6-6fc1-44d6-9d9f-1b9fbb83e07a.png)

### Expected behavior

The name of the 50´s SP Exit corridor should be displayed as "4: Left Turn" 

### Steps to reproduce

Open BEE2.4 and go to "compile options" and there you´ll see this corridor named "right" when it is actually the "left".

**Comment by slambo311** [comment]
Shouldn't this be in BEE2-items?

**Comment by Luke18033** [comment]
Yeah, GitHub isn't letting me transfer it though for some reason? @LautaroL20 Can you try?

**Comment by LautaroL20** [comment]
Alright.

**Comment by LautaroL20** [comment]
Nope, it looks like that Github is kinda buggy and it would not let me transfer it either...
